defaults/main.yml

```diff
@@ -27,8 +27,15 @@ superset_force_restart: False
 
 # white labelling
 superset_white_label: False
+superset_white_label_use_base64: True
+superset_white_label_use_filepaths: False
 
-# full file paths to superset branding images
+# base64 encoded superset branding images
+superset_favicon_base64: REPLACE
```

Comment:
I think this should be an empty string or just nothing (I think that's syntactically correct) instead of `REPLACE` for here and below (to line 41)